src/Serializable.swift

```diff
@@ -148,4 +149,8 @@ public class Serializable: NSObject {
         let (_, some) = mi.children.first!
         return some
     }
+    
+    func isEnum(any : Any) -> Bool {
```

Comment:
Colon Violation: Colons should be next to the identifier when specifying a type. (colon)